eHealthAfrica/aether
# Aet ui push to kernel

#### What does this PR do?
Implements the pipeline publish api endpoint

- Transforms pipeline data into kernel objects (project, schema, mapping)
- Checks if the pipeline objects have links to existing kernel objects (if yes, to be handled in an upcoming PR; if no, creates a new kernel object and save reference to the object on the UI pipeline)

#### How to test
Make a `GET` request `/ui/publish/<pipeline_id>/<project_name>` `pipeline_id` : Id of a pipeline with valid mappings
On kernel, make a submission with the mapping created, Entities should be generated as expected

#### Related JIRA Tickets
[AET-220](https://jira.ehealthafrica.org/browse/AET-220)
[AET-222](https://jira.ehealthafrica.org/browse/AET-222)
[AET-223](https://jira.ehealthafrica.org/browse/AET-223)
